src/test/java/io/cdap/delta/bigquery/BigQueryEventConsumerTest.java

```diff
@@ -482,6 +482,46 @@ public void testSoftDeletes() throws Exception {
     }
   }
 
+  @Test
+  public void testBigQuerySchemaNormalization() throws Exception {
+    String bucketName = "bqtest-" + UUID.randomUUID().toString();
+    Bucket bucket = storage.create(BucketInfo.of(bucketName));
+
+    BigQueryEventConsumer eventConsumer = new BigQueryEventConsumer(MockContext.INSTANCE, storage, bigQuery, bucket,
+        project, 0, STAGING_TABLE_PREFIX, false, null, null,
+        null, false);
+
+    String dataset = "testSchemaNormalization";
+    String tableName = "test-table";
+    String normalizedTableName = "test_table";
+    String fieldName = "field-with-dash";
+    String normalizedFieldName = "field_with_dash";
+
+    try {
+      Schema schema = Schema.recordOf(tableName,
+          Schema.Field.of(fieldName, Schema.of(Schema.Type.INT)));
+      bigQuery.create(DatasetInfo.newBuilder(dataset).build());
+      DDLEvent createTable = DDLEvent.builder()
+          .setOperation(DDLOperation.Type.CREATE_TABLE)
+          .setDatabaseName(dataset)
+          .setTableName(tableName)
+          .setSchema(schema)
+          .setPrimaryKey(Collections.singletonList(fieldName))
+          .setOffset(new Offset())
+          .build();
+      eventConsumer.applyDDL(new Sequenced<>(createTable, 0));
+
+      Table table = bigQuery.getTable(TableId.of(dataset, normalizedTableName));
+      Assert.assertNotNull(table);
+      FieldList bqFields = table.getDefinition().getSchema().getFields();
+      Field normalizedField = bqFields.get(normalizedFieldName);
+      Assert.assertNotNull(normalizedField);
+      Assert.assertEquals(LegacySQLTypeName.INTEGER, normalizedField.getType());
+    }finally {
```

Comment:
nit - add a space after `}`, checkstyle will fail. Please run `mvn clean package -DskipTests` to ensure the checkstyle passes

Reply:
space added.